meerkat_frontend/src/js/explore/explore.js

```diff
@@ -242,32 +242,37 @@ function timelineLink(id, name, axis){
     :param string cat:
         The given category ID.
     :param object options:
-        The options object detailing how the table should be put together. 
+        The options object detailing how the table should be put together.
         Can have the following properties:
-        
+
         * **start_date** (string) - The start date by which to filter data (in ISO format).
         * **end_date** (string) - The end date by which to filter data (in ISO format).
         * **location** (number) - The location ID by which to filter the data. 
         * **strip** (boolean) - Strip empty rows from the table.
         * **colour** (boolean) - Colour the cells with a shade of the highlight colour.
           according to the proportion of the range that the value represents.
+        * **location_case** (boolean) - category breakdown for a given location (as opposed to for a given variable from another category)
 */
 function createTimeline(id, cat, options, title){
 
-	console.log( "Drawing timeline" );
-
 	//These variable will hold all the JSON data from the api, when the AJAX requests are complete.
 	var queryData, category, variable;
 
-	//Assemble the main query url according to the given options.
-	var main_query_url = api_root + '/query_variable/' + id + '/' + cat;
-	if( options.start_date && options.end_date ){
-		main_query_url += '/' + options.start_date + '/' + options.end_date;
-	}
-	main_query_url += '?use_ids=1';
-	if( options.location ){
-		main_query_url += '&only_loc=' + options.location;
-	}
+    //Assemble the main query url according to the given options.
+    var main_query_url = api_root;
+    if(options.location_case){
+        main_query_url += "/query_variable/tot_1/" + cat;
+    }else{
+        main_query_url += '/query_variable/' + id + '/' + cat;
+    }
+
+    if( options.start_date && options.end_date ){
+        main_query_url += '/' + options.start_date + '/' + options.end_date;
+    }
+    main_query_url += '?use_ids=1';
+    if( options.location ){
+        main_query_url += '&only_loc=' + options.location;
+	  }
```

Comment:
Maybe just do a find and replace on all \t tab characters to replace with 4space hard tabs?

Reply:
autoformat does the trick